### PR TITLE
Update Windows install instructions

### PR DESCRIPTION
**Pull request by joa-quim**
Simplify. My new installer comes with ready to work ffmpeg and includes the fix that allow creating movies (rc2 does not).

Oops, sorry. I deleted also the ghostcript install instructions

### CODE REVIEW

**Comment by joa-quim** [comment]
Ping @GenericMappingTools/core 
Better not loose time with this.

File: INSTALL.md [hunk 1]
```diff
@@ -126,35 +126,14 @@ To install a terminal with a bash shell on Windows:
...
-### Using the Windows installer
-
-1. Download and install the GMT windows installer `gmt-6.0.0rc2-win64.exe` from the
-   [GMT 6.0.0rc2 release page](https://github.com/GenericMappingTools/gmt/releases/tag/6.0.0rc2)
-2. Download and install Ghostscript `gs926aw64.exe` [from the official website](https://github.com/ArtifexSoftware/ghostpdl-downloads/releases/tag/gs926)
-3. Download and install ffmpeg by [following these instructions](https://www.wikihow.com/Install-FFmpeg-on-Windows)
-   (including "Enabling FFmpeg in the Command Line")
-
-
-## Building GMT from source
```
Comment (seisman) [inline INSTALL.md]: The `Building GMT from source` section is mainly for Linux users. We should keep it.

Comment (joa-quim) [reply INSTALL.md]: Yes, sure. I think I restored that. It's a bit confusing doing these edits directly on the Github page.

**Comment by leouieda** [comment]
OK, I'll just do some minor style edits and we can merge this

**Review by leouieda — APPROVED** [review]

**Comment by seisman** [comment]
@joa-quim The download speed for the new Windows installer is too slow (less than 100k/s for me). Is it possible to share it via Dropbox publicly?

**Comment by joa-quim** [comment]
https://www.dropbox.com/s/ncgs0d9h5uqtep5/gmt-6.0.0_2019.07.11-win64.exe?dl=0

**Pull request merged**